### PR TITLE
Update userguide with table of contents, quickstart, viewing help and adding clients section

### LINKED ISSUE #15: UG update: Move “Introduction”, “Quickstart”, “Viewing help”, and “Adding clients” sections
- [ ] Introduction
- [ ] Quickstart
- [ ] Viewing Help
- [ ] Adding Clients

### PR DESCRIPTION
**Pull request by LeeEnHao**
Fixes #15

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,36 +5,45 @@ title: User Guide
...
+## Quickstart
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java 11
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the application file (.jar file) from [here](#https://github.com/AY2021S1-CS2103T-F11-4/tp/releases)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Double-click the file to start the app
```
Comment (qwoprocks) [inline docs/UserGuide.md]: I think we should retain the step of copying our jar file over to the folder the user wants to use as the home folder, as we'll need to do saving in a data folder as well

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,36 +5,45 @@ title: User Guide
...
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Type any command in the command box and press Enter to execute it. e.g. typing [`help`](#Viewing-help-help) and
+pressing Enter will open the help window
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+5. (Recommended) The [`help`](#Viewing-help-help) command will show a quick page of some commonly used commands to get
+you started in using TBM
```
Comment (qwoprocks) [inline docs/UserGuide.md]: There's inconsistency between this and the description of the help command below. Also, I think the help contents will be opened in a window and not a page?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -58,29 +67,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
+### Viewing help: `help` 
 
-![help message](images/helpMessage.png)
+Shows a message explaining how to access the help page.
```
Comment (qwoprocks) [inline docs/UserGuide.md]: Refer to comment above

File: docs/UserGuide.md [hunk 4]
```diff
@@ -58,29 +67,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 Format: `help`
 
+### Adding a client: `client add` 
 
-### Adding a person: `add`
+Adds a new client to the app.
 
-Adds a person to the address book.
+Format: `client add n/NAME p/PHONE e/EMAIL a/ADDRESS c/COUNTRY tz/TIMEZONE`
```
Comment (qwoprocks) [inline docs/UserGuide.md]: Maybe we should change PHONE to PHONE_NUMBER, following the original format, for clarity

**Review by qwoprocks — CHANGES_REQUESTED** [review]

**Review by LeeEnHao — COMMENTED** [review]
Removed windows or pages. Left to implementation at later stage

File: docs/UserGuide.md [hunk 5]
```diff
@@ -58,29 +66,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
+### Viewing help: `help` 
 
-![help message](images/helpMessage.png)
+Shows a available commands for TBM.
```
Comment (qwoprocks) [inline docs/UserGuide.md]:
```suggestion
Shows commonly used commands for TBM.
```

**Review by qwoprocks — CHANGES_REQUESTED** [review]

**Review by qwoprocks — APPROVED** [review]

**Pull request merged**